spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ExperimentControllerHelperService.java

```diff
@@ -111,7 +102,7 @@ public JSONObject lookupKitByLotNumber(HttpSession session, JSONObject json) {
     try {
       if (json.has("lotNumber")) {
         String lotNumber = json.getString("lotNumber");
-        Kit kit = requestManager.getKitByLotNumber(lotNumber);
+        KitComponent kit = requestManager.getKitByLotNumber(lotNumber);
```

Comment:
This again comes back to whether the kit components have different lot numbers between bottles in a single kit box, or if every item in the kit has the same lot number. Even so, if each KitComponent represents a single bottle, lotNumber won't be unique so you may wish to be using KitDescriptor instead of KitComponent (or alternatively, add the multiple KitComponents associated with a single KitDescriptor).